dbfit-java/build.gradle

```diff
@@ -83,7 +83,80 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task deploymentProperties {
+        doLast {
+            ext {
+                jdbcDriverClass = ''
+                jdbcDriverUrlPrefix = ''
+                delimeterType = 'normal'
+                delimeter = ';'
+                setUpTestDbScript = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/create-databases-${project.name}.sql"
+                tearDownTestDbScript = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/drop-databases-${project.name}.sql"
+            }
+        }
+    }
+
+    // to be overriden when needed
+    ext.connectionPropertiesFile = '/dev/null'
+    ext.connectionPropertiesFiles = [connectionPropertiesFile]
+
+    task connectionFileProperties {
+        doLast {
+            copyConnectionProperties project.connectionPropertiesFiles, '.'
+            def props = new Properties()
+            file(connectionPropertiesFile).withInputStream { props.load(it) }
+
+            ext {
+                userName = props.getProperty('username')
```

Comment:
Looks like props can be also accessed via alternative syntaxes, e.g.: `props['username']` or `props.username`.

Another wondering is how would it look like if we wrap these in a single ext property of type *map* instead of separate independent string values. (That may make some param lists and assignments shorter. But I'm not sure about all side effects).